src/api.ts

```diff
@@ -13,14 +13,15 @@ import { l } from './logger';
 import { pendingHashes } from './builder';
 import { exec } from 'child_process';
 
-import { CONTAINER_EXPIRY_TIME, START_TIME, TEN_MINUTES } from './constants';
+import { CONTAINER_EXPIRY_TIME } from './constants';
 import { timing } from './stats';
 
 type APIState = {
-	accesses: Map< CommitHash, number >;
+	accesses: Map< ContainerName, number >;
 	branchHashes: Map< CommitHash, BranchName >;
 	containers: Map< string, Docker.ContainerInfo >;
-	localImages: Map< string, Docker.ImageInfo >;
+	localImages: Map< ImageName, Docker.ImageInfo >;
```

Comment:
The data type is equivalent (`ImageName` is an alias for `string`), but the content has changed: before it held only images built by dserve, now it holds a list of all images in the local docker daemon (i.e includes pulled images)